gcc/rust/util/rust-buffered-queue.h

```diff
@@ -72,7 +72,6 @@ template <typename T, typename Source> class buffered_queue
 		  end = num_queued_items;
 		  // TODO: would move be better here? optimisation for move with
 		  // shared pointer?
-
```

Comment:
can you remove this empty line deletion as it separates the TODO comment with the comment on the next line

Reply:
@philberty , ok I will fix it.